config/migrations/postgres/000003_update_configurations_sha.up.sql

```diff
@@ -0,0 +1,25 @@
+ALTER TABLE Configurations ADD COLUMN IF NOT EXISTS SHA VARCHAR(64) DEFAULT '';
+
+CREATE INDEX IF NOT EXISTS idx_configurations_sha ON Configurations(SHA);
+
+DO $$
+	<< migrate_configuration_sha >>
+DECLARE
+	sha_not_exist boolean := FALSE;
+BEGIN
+	SELECT
+		count(*) > 0 INTO sha_not_exist
+	FROM
+		Configurations
+	WHERE
+		SHA = '';
+	IF sha_not_exist THEN
+	UPDATE
+			Configurations
+		SET
+			SHA = SHA256(Value)
```

Comment:
Just realized that this requires postgres 11 and above, maybe we will need to this manually.

Reply:
Yes, we need to do this manually